src/Compilers/Core/Portable/Operations/ConversionKind.cs

```diff
@@ -31,6 +31,11 @@ public enum ConversionKind
         /// </summary>
         OperatorMethod = 0x5,
         /// <summary>
+        /// Conversion is defined by the underlying type, and is created when an interpolated
+        /// string expression is being converted to an IFormattable or FormattableString.
+        /// </summary>
+        InterpolatedString = 0x6,
```

Comment:
>InterpolatedString = 0x6, [](start = 8, length = 25)

Looking at the set of members in this enum, I am not sure if we should be adding InterpolatedString as a member. Should we simply use Basic/CSharp kind instead? For example, we don't try to be more specific about other conversions.